qutebrowser/browser/downloads.py

```diff
@@ -364,7 +384,6 @@ def cancel(self, remove_data=True):
         self.finished.emit()
         self.data_changed.emit()
 
-    @pyqtSlot()
     def delete(self):
```

Comment:
Not your fault, but I think this should also be called `_delete` instead of `delete` because it's not used outside of `DownloadItem`.